values-development.yaml

```diff
@@ -135,14 +135,18 @@ clusterGroup:
 #      project: spring-petclinic
 #      path: charts/region/stage/spring-petclinic-config
 
-    
+
     dev-spring-petclinic:
       name: dev-spring-petclinic
       namespace: devsecops-dev
       project: spring-petclinic
       kustomize: true
       path: charts/region/spring-petclinic/environments/dev
 
+  sharedValueFiles:
+    - '/cluster-overrides/values-{{ $.Values.global.clusterPlatform }}.yaml'
+E   - '/cluster-overrides/values-{{ $.Values.global.clusterPlatform }}-{{ $.Values.global.clusterVersion }}.yaml' #   E:
```

Comment:
Typo?
```suggestion
   - '/cluster-overrides/values-{{ $.Values.global.clusterPlatform }}-{{ $.Values.global.clusterVersion }}.yaml' #   E:
```

Reply:
Yes. An artifact from copy-paste in vim. Fixed.